gui/system/forms.py

```diff
@@ -1029,6 +1030,9 @@ def middleware_clean(self, update):
         for key in ('ui_address', 'ui_v6address'):
             if not update.get(key):
                 update.pop(key, None)
+
+        update['crash_reporting'] = update['crash_reporting'] == 'True'
```

Comment:
Why True as string?

Reply:
`CheckboxInput` for `NullBooleanField` returns it this way for some reason